### PR TITLE
GS1 gtin8 license credential

### PR DESCRIPTION
**Pull request by nissimsan**
From https://ref.gs1.org/gs1/vc/data-model/

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

File: docs/openapi/components/schemas/credentials/GS18PrefixLicenseCredentialyml [hunk 1]
```diff
@@ -0,0 +1,116 @@
+$linkedData:
+  term: GS18PrefixLicenceCredential
+  '@id': https://w3id.org/traceability#GS18PrefixLicenceCredential
+title: GS1 Prefix Licence Credential
+tags:
+  - eCommerce
+description: >-
+  A GS1 Prefix is issued by GS1 Global Office and allocated a GS1 Member Organisation or to itself for the purpose of issuing GS1 Company Prefix or GS1 identification key licences.
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/GS18PrefixLicenseCredentialyml:8]:
```suggestion
  A GS1 Prefix is issued by GS1 Global Office and allocated to a GS1 Member Organization or to itself for the purpose of issuing GS1 Company Prefix or GS1 identification key licenses.
```

File: docs/openapi/components/schemas/credentials/GS18PrefixLicenseCredentialyml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        const:
+          - Organization
+        default:
+          - Organization
+        items:
+          type: string
+          enum:
+            - Organization
+      id:
+        description: The DID for the GS1 Member Organisation, or GS1 Global Office itself, to whom the GS1-8 Prefix has been licensed.
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/GS18PrefixLicenseCredentialyml:70]:
```suggestion
        description: The DID for the GS1 Member Organization, or GS1 Global Office itself, to which the GS1-8 Prefix has been licensed.
```

**Review by mkhraisha — CHANGES_REQUESTED** [review]
I think the file extension isn't working? you may have put `yml` instead of `.yml`

**Comment by nissimsan** [comment]
@nissimsan to merge offline once the file extension bug is fixed.

**Review by mkhraisha — APPROVED** [review]
Assuming Extension changed

**Comment by paulfdietrich** [comment]
@nissimsan.  I did verify against the data model and this looks correct.

**Comment by nissimsan** [comment]
Fixed. Good catch, @mkhraisha - thanks!
Merging.

**Pull request merged**